src/fetch/period_slippage.py

```diff
@@ -5,16 +5,43 @@
 from src.dune_analytics import DuneAnalytics, QueryParameter
 from src.file_io import File
 from src.models import Network
+from src.token_list import HOSTED_ALLOWED_BUFFER_TRADING_TOKEN_LIST_URL, get_trusted_tokens_from_url
+
+
+def generate_sql_query_for_allowed_token_list(token_list) -> str:
+    query = "allow_listed_tokens as ( Select * from (VALUES"
+    for address in token_list:
+        query = "".join([query, f"('\\{address[1:]}' :: bytea),"])
```

Comment:
Inside this loop, it looks like you are putting the name of the sub query in each row.

Reply:
This is an appropriate place to do query += as regular concatenation.

Reply:
Haha, I used join, as I thought you would prefer it. Okay, I will change. I looked it up, it does not matter syntax wise and its only a style question